episodes/formative-assessment.md

```diff
@@ -155,6 +182,66 @@ they might have misconceptions that cause them to arrive at an answer that is in
 but not represented in the MCQ options.
 
 ::::::::::::::::::::::::::::::::::::::::
+                    
+### Example MCQs
+
+#### Coding MCQ
+
+Thinking back of the misconceptions of how values are assigned, referenced and updated in programming languages, 
+here is an example MCQ that can probe learners for such misconceptions. 
+
+_MCQ: Look at the following 3 assignment statements in Python._
+
+```python
+n = 300
+m = n
+n = -100
+```
+
+_What is the result at the end of the above assignments?_
+
+a) n = 300 and m = 300
+b) n = -100 and m = 300
+c) n = -100 and m = -100
+d) n = 200 and m = -100
+
+The correct answer is b., while plausible distractors identify different misconceptions:
+
+a\) they understand the value held by `n` is now also held by `m` but do not understand the value of `n` has 
+been updated since the initial assignment (e.g. treating it as a constant).
+c\) they do not understand `n` and `m` are separate variables/containers for values (which can hold the same values) 
+but rather assume that `m` is referencing variable `n` instead of copying its value at the time of assignment 
+(e.g. behaving as a reference to another cell in a spreadsheet).
+d\) they think that the reassignment for `n` actually subtracts `100` from the original value.
+
+#### Copyright / Licencing MCQ
+
+An example MCQ to check on people's misconceptions around licencing and reusing other people's work
+could be designed as follows.
+
+_MCQ: Which of the following statements are true and which are false?_
+   
+a) I don’t need permission because I am only using the copyrighted work in educational or non-profit purposes  
+b) I should always know the licence of any code, data, libraries, pictures or other work that you reuse or redistribute  
+c) Since I’m planning to give credit to the authors who created the work I reuse, I do not have to worry about or need permission  
+d) Material I obtain from the Internet is publicly accessible so no explicit permission is required  
+e) I don’t need permission because I am only using a small portion of the copyrighted work
+f) The work I want to use does not have a copyright notice on it, so it’s not protected by copyright and I’m free to use it
+g) The work I want to use does not have a licence attached to it, so it’s not protected by copyright and I’m free to use it
```

Comment:
```suggestion
_MCQ: Which of the following statements are true and which are false?_
   
a) I don’t need permission because I am only using the copyrighted work in educational or non-profit purposes  
b) I should always know the licence of any code, data, libraries, pictures or other work that you reuse or redistribute  
c) Since I’m planning to give credit to the authors who created the work I reuse, I do not have to worry about or need permission  
d) Material I obtain from the Internet is publicly accessible so no explicit permission is required  
e) The work I want to use does not have a copyright notice on it, so it’s not protected by copyright and I’m free to use it
```

Reply:
Addressing a comment by @tobyhodges about cutting down the number of options in the copyright/licence  MCQ